server/app/featureflags/FeatureFlags.java

```diff
@@ -42,6 +42,9 @@ public final class FeatureFlags {
 
   // Common Intake Form flags.
   private static final String INTAKE_FORM_ENABLED = "intake_form_enabled";
+
+  // phone number question type
```

Comment:
**nitpick:** The nongated eligibility flag is part of the common intake form flags. Could you move the phone question type flag below that one please? Here and below.